test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -652,7 +652,6 @@ private void RestartTestSetup(bool startInTestMode)
 
             //create the view
             View = new DynamoView(ViewModel);
-            View.Show();
```

Comment:
`RestartTestSetup` is only used in one test which requires the test mode to be turned off but does not require the view to be loaded.